packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -303,13 +303,30 @@ class Scrollable extends StatefulWidget {
   /// Calling this method will create a dependency on the closest [Scrollable]
```

Comment:
nit: This is ever so slightly inaccurate if you pass in an axis because what you depend on may not be the _closest_ Scrollable...